upgrade_test.py

```diff
@@ -297,6 +357,9 @@ def test_rolling_upgrade(self):
         self.log.info('Sleeping for 60s to let cassandra-stress start before the upgrade...')
         time.sleep(60)
 
+        # Prepare keyspace and tables for truncate test
+        self.fill_db_data_for_truncate_test()
```

Comment:
Hi @juliayakovlev How much test time is increased by truncate test?

Reply:
Create 100 tables and insert 100 rows in each table is taken about 30 min.
Truncating is fast

Reply:
Changed to 10 tables with 100 record each table